src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtil.java

```diff
@@ -44,7 +44,7 @@ public static Scope getScopeFromMods(DetailAST aMods) {
         Scope returnValue = Scope.PACKAGE;
         for (DetailAST token = aMods.getFirstChild(); token != null
                 && returnValue == Scope.PACKAGE;
-                token = token.getNextSibling()) {
+             token = token.getNextSibling()) {
```

Comment:
Now the indent is a single space. Please revert or reformat this expression completely, may be
```java
for (DetailAST token = aMods.getFirstChild();
        token != null && returnValue == Scope.PACKAGE;
        token = token.getNextSibling()) {
    if ("public".equals(token.getText())) {
```
?

Reply:
Fixed.